src/Homework4/task2.py

```diff
@@ -0,0 +1,66 @@
+"""Города
+   Дан список стран и городов каждой страны. Затем даны названия городов.
+   Для каждого города укажите, в какой стране он находится.
+   Входные данные
+   Программа получает на вход количество стран N. Далее идет N строк, каждая
+   строка начинается с наз-я страны, затем идут наз-я городов этой страны.
+   В следующей строке записано число M, далее идут M запросов — названия
+   каких-то M городов, перечисленных выше.
+   Выходные данные
+   Для каждого из запроса выведите название страны, в котором находится
+   данный город.
+   Примеры
+   Входные данные
+   2
+   Russia Moscow Petersburg Novgorod Kaluga
+   Ukraine Kiev Donetsk Odessa
+   3
+   Odessa
+   Moscow
+   Novgorod
+   Выходные данные
+   Ukraine
+   Russia
+   Russia
+"""
+
+
+n = int(input('Введите кол-во стран: '))
+for i in range(n):
+    capitals = input('Введите ' + str(i + 1) + ' -ю страну и ее города: ')
+    a = capitals.split()
+m = int(input('Введите кол-во городов: '))
+for j in range(m):
+    cities = input('Введите ' + str(j + 1) + ' -й город: ')
+    for k in cities:  # ИСПРАВИТЬ !!! не работает
```

Comment:
comments

Reply:
yes, I have not yet understood how))) this is not to forget